core/src/main/scala/org/broadinstitute/dsde/rawls/spendreporting/SpendReportingService.scala

```diff
@@ -571,28 +574,87 @@ class SpendReportingService(
           return Future.successful(None)
         }
         projectNames: Map[GoogleProjectId, WorkspaceName] = billingMap.values.flatten.toMap
+        projectIds = projectNames.keySet.map(x => x.value)
```

Comment:
nit: you don't need an `x` variable here:
```suggestion
        projectIds = projectNames.keySet.map(_.value)
```